config/initializers/job_configurations.rb

```diff
@@ -199,6 +199,12 @@
         cron: cron_1w,
         args: -> { [Time.zone.now] },
       },
+      # Send fraud metrics to Team Judy
+      fraud_metrics_report: {
+        class: 'Reports::FraudMetricsReport',
+        cron: cron_24h,
+        args: -> { [Time.zone.now] },
```

Comment:
match the MKMR, and also the initializer
```suggestion
        args: -> { [Time.zone.yesterday.end_of_day] },
```